frontend-web/src/coffee-call/coffee-call.tsx

```diff
@@ -12,14 +12,20 @@ export const CoffeeCall: Component = () => {
 
   const username = getUsername();
 
+  const newestMessage = from(protocol.messages);
+  const allMessages = createMemo((oldValue: CallMessage[]) => {
+    const actualNewMessage = newestMessage();
+    return actualNewMessage === undefined ? [...oldValue] : [...oldValue, actualNewMessage]
+  }, []);
```

Comment:
The name `actualNewMessage` really annoys me.  
How about this:

```suggestion
  const messageSignal = from(protocol.messages);
  const messages = createMemo((existingMessages: CallMessage[]) => {
    const message = messageSignal();
    return message === undefined ? existingMessages : [...existingMessages, message]
  }, []);
```

And yes, it is safe to not re-create the existing message array for `undefined` (would be an empty array anyway).

Note that applying the suggestion also needs a rename in the `<For>`, see next suggestion.